### PR TITLE
Bringing react template up to level

### PR DESCRIPTION
**Pull request by nickbalestra**
beta release
- [x] SSR
- [x] CSS-Modules
- [x] Scaffold
- [x] Webpack configurator now target dev/production
- [x] tests

### CODE REVIEW

**Comment by nickbalestra** [comment]
It's a start.

```
----------------------------------------------------------------|----------|----------|----------|----------|----------------|
File                                                            |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------------------------------------------------------------|----------|----------|----------|----------|----------------|
All files                                                       |     91.3 |     74.7 |    92.86 |    91.57 |                |
 oc-template-react                                              |      100 |      100 |      100 |      100 |                |
  index.js                                                      |      100 |      100 |      100 |      100 |                |
 oc-template-react-compiler                                     |      100 |      100 |      100 |      100 |                |
  index.js                                                      |      100 |      100 |      100 |      100 |                |
 oc-template-react-compiler/lib                                 |    90.57 |       70 |     93.1 |    90.45 |                |
  compile.js                                                    |     92.5 |       90 |      100 |    92.31 |       56,57,58 |
  compileServer.js                                              |    84.62 |     62.5 |       50 |    84.62 |    59,60,64,68 |
  compileStatics.js                                             |    83.33 |    55.56 |      100 |    83.33 |... 41,64,65,66 |
  compileView.js                                                |    96.49 |    78.57 |      100 |    96.43 |         45,103 |
 oc-template-react-compiler/lib/to-abstract-base-template-utils |    92.68 |    83.33 |      100 |    92.68 |                |
  compiler.js                                                   |    85.71 |       70 |      100 |    85.71 |       23,34,40 |
  webpackConfigurator.js                                        |      100 |    92.86 |      100 |      100 |             86 |
 oc-template-react-compiler/lib/to-be-published                 |      100 |      100 |      100 |      100 |                |
  oc-react-component-wrapper.js                                 |      100 |      100 |      100 |      100 |                |
 oc-template-react/lib                                          |    90.63 |       75 |       80 |    93.55 |                |
  render.js                                                     |    90.48 |      100 |    66.67 |       95 |             33 |
  try-get-cached.js                                             |    90.91 |       75 |      100 |    90.91 |             15 |
 oc-template-react/lib/to-be-published                          |       90 |    66.67 |      100 |    88.89 |                |
  get-js-from-url.js                                            |       90 |    66.67 |      100 |    88.89 |             14 |
----------------------------------------------------------------|----------|----------|----------|----------|----------------|
```

File: package.json [hunk 1]
```diff
@@ -1,91 +1,30 @@
...
-    }
+  "devDependencies": {
+    "husky": "^0.14.3",
+    "codecov": "^2.2.0",
+    "jest": "^20.0.4",
+    "lerna": "^2.0.0",
+    "lint-staged": "^4.0.3",
+    "prettier-eslint-cli": "^4.2.0"
   },
-  "compileDependencies": {
```
Comment (nickbalestra) [inline package.json]: :) those earlier API

File: packages/oc-template-react-compiler/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# oc-template-handlebars-compiler - Compiler module
```
Comment (danrr) [inline packages/oc-template-react-compiler/README.md]: handlebars => react?

**Comment by nickbalestra** [comment]
@matteofigus This should be ok for a very first beta0.

**Pull request merged**